templates/company-board/test/company-board.js

```diff
@@ -59,98 +59,90 @@ contract('Company with board', ([_, owner, boardMember1, boardMember2, shareHold
     template = CompanyTemplate.at(address)
   })
 
-  const setupShare = (...params) => {
+  const finalizeInstance = (...params) => {
     const lastParam = params[params.length - 1]
     const txParams = (!Array.isArray(lastParam) && typeof lastParam === 'object') ? params.pop() : {}
-    const setupShareFn = CompanyTemplate.abi.find(({ name, inputs }) => name === 'setupShare' && inputs.length === params.length)
-    return template.sendTransaction(encodeCall(setupShareFn, params, txParams))
+    const finalizeInstanceFn = CompanyTemplate.abi.find(({ name, inputs }) => name === 'finalizeInstance' && inputs.length === params.length)
+    return template.sendTransaction(encodeCall(finalizeInstanceFn, params, txParams))
   }
 
   context('when the creation fails', () => {
     const FINANCE_PERIOD = 0
     const USE_AGENT_AS_VAULT = true
 
     context('when there was no instance prepared before', () => {
-      it('reverts when no board members are given', async () => {
-        await assertRevert(template.prepareInstance(SHARE_TOKEN_NAME, SHARE_TOKEN_SYMBOL, [], BOARD_VOTING_SETTINGS), 'COMPANY_MISSING_BOARD_MEMBERS')
```

Comment:
I tried testing

```js
await assertRevert(template.prepareInstance(SHARE_TOKEN_NAME, SHARE_TOKEN_SYMBOL, [], BOARD_VOTING_SETTINGS), 'COMPANY_MISSING_BOARD_MEMBERS')
```

But the settings don't seem to work very well (it appears their length is always allocated to the expected amount)?

Reply:
Yea, exactly